<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTextOperateWidget</class>
 <widget class="QWidget" name="CTextOperateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_tool" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_tool1" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
         <property name="spacing">
          <number>-1</number>
         </property>
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFontComboBox" name="fontComboBox">
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_fontSize">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_tool2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton_bold">
           <property name="toolTip">
            <string>黑体</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Wordnote2.qrc">
             <normaloff>:/images/text_bold_black.svg</normaloff>:/images/text_bold_black.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_italic">
           <property name="toolTip">
            <string>斜体</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Wordnote2.qrc">
             <normaloff>:/images/text_italic_black.svg</normaloff>:/images/text_italic_black.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_underline">
           <property name="toolTip">
            <string>下划线</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Wordnote2.qrc">
             <normaloff>:/images/text_underline_black.svg</normaloff>:/images/text_underline_black.svg</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_color" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton_fontColor">
           <property name="toolTip">
            <string>字体颜色</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Wordnote2.qrc">
             <normaloff>:/images/text_color_black.svg</normaloff>:/images/text_color_black.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_fontBackgroundColor">
           <property name="toolTip">
            <string>荧光笔</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Wordnote2.qrc">
             <normaloff>:/images/text_background_color_black.svg</normaloff>:/images/text_background_color_black.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_align" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton_alignLeft">
           <property name="toolTip">
            <string>左对齐</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Wordnote2.qrc">
             <normaloff>:/images/text_align_left_black.svg</normaloff>:/images/text_align_left_black.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_alignCenter">
           <property name="toolTip">
            <string>中间对齐</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Wordnote2.qrc">
             <normaloff>:/images/text_align_center_black.svg</normaloff>:/images/text_align_center_black.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_alignRight">
           <property name="toolTip">
            <string>右对齐</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Wordnote2.qrc">
             <normaloff>:/images/text_align_right_black.svg</normaloff>:/images/text_align_right_black.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_alignBothEnds">
           <property name="toolTip">
            <string>两端对齐</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Wordnote2.qrc">
             <normaloff>:/images/text_align_justify_black.svg</normaloff>:/images/text_align_justify_black.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_forwardBack" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton_backward">
           <property name="toolTip">
            <string>撤回</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Wordnote2.qrc">
             <normaloff>:/images/text_backward_black.svg</normaloff>:/images/text_backward_black.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_forward">
           <property name="toolTip">
            <string>前进</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Wordnote2.qrc">
             <normaloff>:/images/text_forward_black.svg</normaloff>:/images/text_forward_black.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_picture">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Wordnote2.qrc">
          <normaloff>:/images/picture_black.svg</normaloff>:/images/picture_black.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_save">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Wordnote2.qrc">
          <normaloff>:/images/text_save_black.svg</normaloff>:/images/text_save_black.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_content" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Wordnote2.qrc"/>
 </resources>
 <connections/>
</ui>
